Exoft.Gamification.Api.Services/RequestAchievementService.cs

```diff
@@ -57,11 +53,14 @@ public async Task<IResponse> AddAsync(RequestAchievementModel model, Guid userId
             pageWithParams = pageWithParams.Replace("{message}", model.Message);
             pageWithParams = pageWithParams.Replace("{achievementName}", achievement.Name);
 
-            await _emailService.SendEmailAsync(
-                _adminContact.Mail,
-                "Request achievement",
-                pageWithParams,
-                true);
+            var emails = await _userRepository.GetAdminsEmailsAsync();
+            foreach (var email in emails)
+            {
+                await _emailService.SendEmailAsync(
```

Comment:
Not sure, but smtp service has send many emails method.... So please extend your email service